app/webpack/users/edit/components/toggle_switch.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const ToggleSwitch = ( { profile, name, handleCheckboxChange } ) => (
+  <div className="col-xs-4 col-md-3 ToggleSwitch">
+    <div className="flex-no-wrap">
+      <label htmlFor={name}>{I18n.t( "off_toggle" ).toLocaleUpperCase( )}</label>
+      <label className="switch">
+        <input
+          name={name}
+          type="checkbox"
+          checked={profile[name] || false}
+          onChange={handleCheckboxChange}
+        />
+        <span className="slider round" />
+      </label>
+      <label htmlFor={name}>{I18n.t( "on_toggle" ).toLocaleUpperCase( )}</label>
+    </div>
+  </div>
+);
+
+ToggleSwitch.propTypes = {
+  profile: PropTypes.object,
```

Comment:
This component is great, but this prop is keeping it from being more generally useful. Maybe replace it with `checked`? The other thing we'd need to do is pull its styles out into a separate file that could be used elsewhere.

Reply:
Replaced with a `checked` prop and moved styles into `users/_toggle_switch` file.